vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/ConjunctionImpl.java

```diff
@@ -95,7 +95,7 @@ public String toString() {
 			} else {
```

Comment:
why don't you call getConjunctionString() inside the toString() method?
currently, toString() does the same thing as getConjunctionString(), so the code is duplicated.

Reply:
Ok, I thought I already removed it